gobblin-data-management/src/test/java/org/apache/gobblin/data/management/copy/iceberg/IcebergRegisterStepTest.java

```diff
@@ -41,13 +43,7 @@ public void testDestSideMetadataMatchSucceeds() throws IOException {
     TableMetadata justPriorDestTableMetadata = mockTableMetadata("foo", "bar");
     TableMetadata readTimeSrcTableMetadata = Mockito.mock(TableMetadata.class); // (no mocked behavior)
     IcebergTable mockTable = mockIcebergTable("foo", "bar"); // matches!
-    IcebergRegisterStep regStep =
-        new IcebergRegisterStep(srcTableId, destTableId, readTimeSrcTableMetadata, justPriorDestTableMetadata, new Properties()) {
-          @Override
-          protected IcebergCatalog createDestinationCatalog() throws IOException {
-            return mockSingleTableIcebergCatalog(mockTable);
-          }
-        };
+    IcebergRegisterStep regStep = createIcebergRegisterStepInstance(readTimeSrcTableMetadata, justPriorDestTableMetadata, mockTable, new Properties());
```

Comment:
thank you - appreciating your improvement stewardship! :)

Reply:
Thank you:)